traffic_ops/testing/compare/compare.go

```diff
@@ -138,7 +138,59 @@ func testRoute(tos []*Connect, route string) {
 	wg.Wait()
 	close(ch)
 
-	if res[0].Res == res[1].Res {
```

Comment:
expected that most lines **will** be identical..   the config file comments with timestamps are a minority,  so why not do the fast string comparison first and skip if the lines are the same?

Reply:
That honestly just comes down to preference - I wanted the comparison to always look at the same variables, so I set their default values as the response bodies, then checked if it was a configfile before proceeding to the comparison.

Reply:
I don't really think putting two assignments and a conditional in front of it will degrade performance in any significant way, but it's trivial to move them around if you'd prefer. Downside is I'd be calling into the `writeResults` function in an additional place

Reply:
ok